src/App/Fossa/Analyze.hs

```diff
@@ -81,9 +62,52 @@ data ScanDestination
   = UploadScan URI ApiKey ProjectMetadata -- ^ upload to fossa with provided api key and base url
   | OutputStdout
 
-analyzeMain :: BaseDir -> Severity -> ScanDestination -> OverrideProject -> Bool -> IO ()
-analyzeMain basedir logSeverity destination project unpackArchives = withLogger logSeverity $
-  analyze basedir destination project unpackArchives
+analyzeMain :: BaseDir -> Severity -> ScanDestination -> OverrideProject -> Bool -> [BuildTargetFilter] -> IO ()
+analyzeMain basedir logSeverity destination project unpackArchives filters = withLogger logSeverity $
+  analyze basedir destination project unpackArchives filters
+
+discoverFuncs ::
+  ( Has (Lift IO) sig m,
+    MonadIO m,
+    Has ReadFS sig m,
+    Has Exec sig m,
+    Has Logger sig m,
+    Has Diag.Diagnostics sig m
+  ) =>
+  -- | Discover functions
+  [Path Abs Dir -> m [NewProject]]
+discoverFuncs =
```

Comment:
Not all of the analysis types have been migrated. That'll happen in the next day or two as I have time to cherry-pick out the commits

Reply:
Have the analyses all been migrated?  Are there any pending (not yet pushed to this PR) that you know of?

Reply:
They're all migrated now